support/Pipelines/Bbh/Inspiral.yaml

```diff
@@ -54,9 +54,13 @@ DomainCreator:
         ExciseWithBoundaryCondition:
           DemandOutgoingCharSpeeds:
       UseLogarithmicMap: true
+    # For the envelope, setting radial distribution to logarithmic produces
+    # lower constraints than other radial distrubtions.
     Envelope:
       Radius: 100.0
       RadialDistribution: Logarithmic
+    # For the outer shell, setting radial distribution to linear produces lower
+    # constraints than other radial distrubtions.
```

Comment:
I suggest something like "Radial distribution is linear to resolve gravitational waves"